compiler/src/org.graalvm.compiler.hotspot.aarch64/src/org/graalvm/compiler/hotspot/aarch64/AArch64HotSpotBackend.java

```diff
@@ -121,7 +121,9 @@ protected void bangStackWithOffset(CompilationResultBuilder crb, int bangOffset)
         AArch64MacroAssembler masm = (AArch64MacroAssembler) crb.asm;
         try (ScratchRegister sc = masm.getScratchRegister()) {
             Register scratch = sc.getRegister();
-            AArch64Address address = masm.makeAddress(sp, -bangOffset, scratch, 8, /* allowOverwrite */false);
+            AArch64Address address = masm.makeAddress(sp, -bangOffset, scratch, 8, /*
```

Comment:
I'm wondering what's up with the formater. If the format was fine before, it should still be ok. Maybe we can simply remove that change.